site/src/examples/border-layout/BorderItemAlignment.tsx

```diff
@@ -0,0 +1,96 @@
+import { ReactElement, useState, ChangeEventHandler } from "react";
+import {
+  BorderLayout,
+  BorderItem,
+  GRID_ALIGNMENT_BASE as borderItemAlignment,
+  RadioButtonGroup,
+  RadioButton,
+  FormField,
+  FormFieldLabel,
+} from "@salt-ds/core";
+import styles from "./index.module.css";
+import borderItemAlignmentStyles from "./BorderItemAlignment.module.css";
+
+const alignmentOptions = borderItemAlignment.map((alignment, index) => ({
+  label: alignment,
+  value: `option${index + 1}`,
+}));
+
+export const BorderItemAlignment = (): ReactElement => {
+  const [verticalOption, setVerticalOption] = useState("option4");
+  const [horizontalOption, setHorizontalOption] = useState("option4");
+
+  const handleVerticalChange: ChangeEventHandler<HTMLInputElement> = (
+    event
+  ) => {
+    const { value } = event.target;
```

Comment:
NIT: it doesn't really make sense to destructure in these cases. Normally it's only done when a linter forces you

Reply:
It does throw an error if I pass `event.target` directly :(
`Argument of type 'EventTarget & HTMLInputElement' is not assignable to parameter of type 'SetStateAction<string>'`